src/eXpansion/Bundle/Dedimania/Resources/config/configs.yml

```diff
@@ -0,0 +1,37 @@
+services:
+    _defaults:
+        autowire: true
+        autoconfigure: true
+
+    expansion.dedimania.config.enable:
```

Comment:
I am not fun of the enable thing. We will have a proper system for it later. can we remove for now?